http/server.lua

```diff
@@ -1290,6 +1290,7 @@ local exports = {
             log_requests        = true,
             log_errors          = true,
             display_errors      = true,
+            keepalive_disable   = {},
```

Comment:
It does not look related to this commit.

Reply:
True, fixed.